common/scripts/Ubuntu_14.04/installVault.sh

```diff
@@ -53,9 +53,17 @@ check_vault() {
 }
 
 main() {
+  check_vault=""
   {
-    type vault &> /dev/null && echo "Vault already installed, skipping" && return
-  }
+    type vault &> /dev/null
```

Comment:
why cant we use the service command here too? we're running vault using upstart so it should be in the service list